tekton/integration_pipeline.go

```diff
@@ -19,16 +19,17 @@ package tekton
 import (
 	"encoding/json"
 	"fmt"
-	"reflect"
-	"strings"
-
 	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
 	"github.com/redhat-appstudio/integration-service/api/v1beta2"
 	"github.com/redhat-appstudio/operator-toolkit/metadata"
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"reflect"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strings"
+	"time"
```

Comment:
IDE fight, I like the original way more

Reply:
Missed this one, reverted it back to how it was.